### DISCUSSION TITLE
Can u add https://mein-abfallkalender.de/home.html

### DISCUSSION THREAD

**Discussion by Ratzi2k** [mampfes/hacs_waste_collection_schedule#803] [Ideas]
Hey,
is it possible to add this source?

Town:
Street:

https://mein-abfallkalender.de/home.html

**Comment by mampfes** [comment]
mein-abfallkalender.de is already supported by the [ICS](https://github.com/mampfes/hacs_waste_collection_schedule/blob/master/doc/source/ics.md) source.

But don't forget to do a complete registration including email! Otherwise it will not work.

**Comment by mampfes** [comment]
Supported and documented by ics source.
